src/battle_ai_main.c

```diff
@@ -3272,6 +3272,13 @@ static u32 AI_CalcMoveEffectScore(u32 battlerAtk, u32 battlerDef, u32 move)
                 ADJUST_SCORE(DECENT_EFFECT);
         }
         break;
+    case EFFECT_FINAL_GAMBIT:
```

Comment:
Could you add final gambit to the function `AI_IsMoveEffectInMinus` and I believe Explosion is missing there as well

Reply:
Sounds good on both accounts! :)

Reply:
no condition. It should be considered a bad move in general. This is just for the comparison for moves

Reply:
Okay gotcha! Added the effects to the list :)

Reply:
ty!